org.osgi.test.common/src/main/java/org/osgi/test/common/install/BundleInstaller.java

```diff
@@ -207,6 +207,12 @@ public int hashCode() {
 
 		public InputStream openStream(BundleContext bc) throws IOException {
 
+			URL jarEmbeddedFileUrl = getURL(bc);
+
+			return jarEmbeddedFileUrl.openStream();
+		}
+
+		public URL getURL(BundleContext bc) throws FileNotFoundException {
```

Comment:
I think `IOException` is a better choice for the throws.

Reply:
it is.